### ISSUE TITLE
gather usage stats for LF to know who are our users

### ISSUE THREAD

**Issue by megahirt** [sillsdev/web-languageforge#823]
There are a number of statistics that we should gather to formally report on.  This task is about a manual process to gather LF usage statistics.  Automating statistics gathering via a dashboard would be lovely and ultimately a nice goal, but we are far off of that.

Send/Receive Stats

- [ ] How many projects using send/receive
- [ ] When they last synced
- [ ] Size of project
- [ ] Frequency of S/R

Activity Stats

- [ ] How many projects are active in a given time period?
- [ ] How many entries edited?

Project Stats

- [ ] Number of projects containing pictures
- [ ] Number of projects containing audio
- [ ] Number of projects containing comments

User Stats

- [ ] User logins
- [ ] Number of active users
- [ ] Projects with more than one active user (how many)

Website Stats (Google Analytics)

- [ ] Number of people accessing our site
- [ ] From where?

Google Play Stats

- [ ] Number of installs, etc.

**Comment by alex-larkin** [comment]
I'm really glad @megahirt that you added this to the To-Do list! I was just thinking over the weekend that this is really quite important- we need to know who our users are and what they need, in order to be able to make an effective tool! I was going to bump it up from last on my list, but you read my mind 😀

**Comment by longrunningprocess** [comment]
Excellent, thank you for creating this

**Comment by longrunningprocess** [comment]
after discussing at our last team meeting we concluded, while valuable for insights and starting conversations, the data I've gathered is not as actionable as we had hoped at this time.  We will need to engage in other efforts before making a real dent in our housekeeping desires.  I'll keep the gathered data on our team share for use another day.

@megahirt I'll leave the final disposition of this issue to you.
